### DISCUSSION TITLE
Open at Startup errors

### DISCUSSION THREAD

**Discussion by s3891667** [nvim-tree/nvim-tree.lua#2049] [Q&A]
After setting up a new codes for open_on_setup, I notice that this error keep displaying when ever I open neovim for 1 second then disappears, does anyone know how to fix it ? It's a bit annoying... 

![Screenshot from 2023-03-13 01-39-26](https://user-images.githubusercontent.com/80374329/224551803-406821e7-e434-422b-9f94-eec5049dd250.png)

**Comment by alex-courtis** [comment]
This might be an issue with other plugins or automation.

Please attempt a [wiki: "Clean Room" Replication](https://github.com/nvim-tree/nvim-tree.lua/wiki/Troubleshooting#clean-room-replication) and raise a bug if needed.
